gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -377,6 +377,21 @@
 		<type>\$type.type</type>
 		<nports>\$nchan</nports>
 	</$sourk>
+#if $sourk == 'source'
+	<source>
+		<name>rx_metad</name>
```

Comment:
Fair point. Will change.

Am 6. Juni 2016 23:43:21 MESZ, schrieb mbr0wn <notifications@github.com>:
>> @@ -377,6 +377,21 @@
>>  		<type>\$type.type</type>
>>  		<nports>\$nchan</nports>
>>  	</$sourk>
>> +#if $sourk == 'source'
>> +	<source>
>> +		<name>rx_metad</name>
>
>That's why GRC has a button to disable port names, and I doubt the 3
>chars make such a h differe.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/gnuradio/gnuradio/pull/835/files/bf7ac1bfa972a5bc97f4d485531e82f6fc277e9b#r65974978

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.